src/library/index.ts

```diff
@@ -90,6 +91,29 @@ export default function (options: LibraryOptions): Rule {
         });
     }
 
+    function updateAngularCompilerOptionsToTsConfig(folderName: string): Rule {
+        return (host: Tree): Tree => {
+            const path = `${folderName}/tsconfig.lib.json`;
+            const file = host.read(path);
+            // Necessary to remove a comment (comments are not accepted when converting a string to JSON)

```

Comment:
Comments in JSON 🤯

Maybe easier to use https://www.npmjs.com/package/strip-json-comments ?
Or maybe even require('typescript').parseConfigFileTextToJson()

Reply:
Or better yet: update the config file via typescript api calls.